test/terraform_aws_alb_web_containers_test.go

```diff
@@ -42,6 +42,69 @@ func TestTerraformAwsAlbWebContainersSimpleHttp(t *testing.T) {
 		},
 	}
 
+	defer terraform.Destroy(t, terraformOptions)
+
+	// This will run `terraform init` and `terraform apply`
+	_, err := terraform.InitAndApplyE(t, terraformOptions)
+
+	/*
+		We need to apply twice because of the current bug with `dynamic` blocks in terraform `aws` provider:
+		Error: Provider produced inconsistent final plan
+
+		When expanding the plan for module.alb.aws_lb.main to include new values
+		learned so far during apply, provider "registry.terraform.io/hashicorp/aws"
+		produced an invalid new value for .access_logs[0].bucket: was
+		cty.StringVal(""), but now cty.StringVal("terratest-rbbbb7-logs").
+
+		This is a bug in the provider, which should be reported in the provider's own
+		issue tracker.
+		https://github.com/terraform-providers/terraform-provider-aws/issues/10297
```

Comment:
This is another bug report specific to AWS access logs and ALBs
https://github.com/hashicorp/terraform-provider-aws/issues/16674